gnd/src/main/java/com/google/android/gnd/model/basemap/tile/Tile.java

```diff
@@ -37,14 +46,16 @@ public static Builder newBuilder() {
     return new AutoValue_Tile.Builder();
   }
 
-  public static String pathFromId(String tileId) {
+  public static String pathFromCoords(int x, int y, int z) {
```

Comment:
Would `getFilenameFromCoords()` be more appropriate?

Reply:
I like filenameFromCoords --but I'm dropping `get` because this method is static (though maybe it shouldn't be static)!